app/uk/gov/hmrc/pensionschemereturn/models/etmp/nonsipp/EtmpMemberPayments.scala

```diff
@@ -45,7 +45,8 @@ case class EtmpMemberDetails(
   pensionAmountReceived: Option[Double],
   personalDetails: EtmpMemberPersonalDetails,
   memberEmpContribution: List[EtmpEmployerContributions],
-  memberTransfersIn: List[EtmpTransfersIn]
+  memberTransfersIn: List[EtmpTransfersIn],
+  memberLumpSumReceived: Option[List[EtmpMemberLumpSumReceived]]
```

Comment:
This one is `Option[List]` while other (e.g. memberEmpContribution) is a `List[]`.
I cannot see any difference in schema, should these be the same?
If so, which one is correct?

Reply:
Except memberStatus and personalDetails, they are all optional according to API schema. But in this case, since list can be empty for memberDetails we can use only List for all if you want.

Reply:
I'd say yes please... but not because I want it, just to be consistent with the rest of the code :)

Reply:
As discussed, if the version with Option is correct, we'd need to create a bug to change the others and we can resolve this.